pulp_puppet_plugins/pulp_puppet/plugins/importers/importer.py

```diff
@@ -80,9 +81,21 @@ def upload_unit(self, repo, type_id, unit_key, metadata, file_path, conduit, con
         try:
             report = upload.handle_uploaded_unit(repo, type_id, unit_key, metadata, file_path,
                                                  conduit)
+
```

Comment:
@daviddavis +1 to what @mhrivnak said. I was going to write the same thing. The message associated with an Exception that bubbles all the way up during task execution is recorded in a way that it will be shown to the user. Since this is a fatal exception I think that is the right behavior.